File: bank-app/app/src/main/java/com/android/bankapp/login/LoginResponse.java
package com.android.bankapp.login;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class LoginResponse {

    @SerializedName("userAccount")
    @Expose
    private UserAccount userAccount;
    @SerializedName("error")
    @Expose
    private Error error;

    public UserAccount getUserAccount() {
        return userAccount;
    }

    public void setUserAccount(UserAccount userAccount) {
        this.userAccount = userAccount;
    }

    public Error getError() {
        return error;
    }

    public void setError(Error error) {
        this.error = error;
    }

    @Override
    public String toString() {
        return "LoginResponse{" +
                "userAccount=" + userAccount +
                ", error=" + error +
                '}';
    }
}

File: app/src/main/java/com/earaujo/santander/repository/models/LoginResponse.kt
package com.earaujo.santander.repository.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class LoginResponse(
    @SerializedName("userAccount")
    @Expose
    var userAccountModel: UserAccountModel?,
    @SerializedName("error")
    @Expose
    var error: ErrorModel?
)